completions/curl

```diff
@@ -153,6 +153,11 @@ _comp_cmd_curl()
             _comp_compgen -- -W 'SRP'
             return
             ;;
+        --trace)
+            _comp_compgen -- -W '%'
```

Comment:
If we do this, should we complete `-` as well (for sending the trace to stdout)?

Also, `--trace-ascii` appears to take `%` and `-` as well even though in my curl man page only the latter is documented.

Reply:
> If we do this, should we complete `-` as well (for sending the trace to stdout)?

Yeah, added 

> Also, `--trace-ascii` appears to take `%` and `-` as well even though in my curl man page only the latter is documented.

Added for that too, it seems `%` was added to the man here: https://github.com/curl/curl/commit/1159dc359da767c36ed724f246cd0a3bd4b1f7ce